### DISCUSSION TITLE
Highlighter hug shapes

### DISCUSSION THREAD

**Discussion by pandika-csgi** [clientIO/joint#1910] [Q&A] [answered]
Hi, is possible to make highlighter "hug" shape when we are try to connect 2 shape?
![Screen Shot 2022-11-28 at 15 38 46](https://user-images.githubusercontent.com/106044262/204220871-dcfe210a-a836-43fc-9002-ef1b6c7553d9.png)
From that picture, I want blue highlighter become rounded and "hug" origin shape

Thank you

**Comment by kumilingus** [comment, marked as answer]
Use [`highlighterSelector` attribute](https://resources.jointjs.com/docs/jointjs/v3.6/joint.html#dia.attributes.highlighterSelector) to specify which part of the `ElementView` you want to highlight.

The highlighters (`stroke` and `mask`) draw the tight outline around the node by default. In case the node is an `SVGGroup` or `SVGText` they draw a rectangle.

Here's an [example](https://jsfiddle.net/kumilingus/8279fud3/).

**Reply by pandika-csgi** [reply]
Thankss 🙌🏽
